openvdb/points/AttributeArray.h

```diff
@@ -1745,7 +1744,7 @@ TypedAttributeArray<ValueType_, Codec_>::readBuffers(std::istream& is)
     std::unique_ptr<char[]> buffer(new char[mCompressedBytes]);
     is.read(buffer.get(), mCompressedBytes);
     mCompressedBytes = 0;
-    mFlags &= ~PARTIALREAD; // mark data read as having completed
```

Comment:
Conversion to `uint8_t` from `int`.

Reply:
Is this a compiler that's picking up this issue, in which case which one out of interest?

Reply:
It was GCC 4.8, IIRC.  But `PARTIALREAD` is an untyped enum, which defaults to `int`, if I'm not mistaken.